jenkins/spark-premerge-build.sh

```diff
@@ -37,10 +37,10 @@ export PATH="$SPARK_HOME/bin:$SPARK_HOME/sbin:$PATH"
 tar zxf $SPARK_HOME.tgz -C $ARTF_ROOT && \
     rm -f $SPARK_HOME.tgz
 
-mvn -U -B $MVN_URM_MIRROR '-Pinclude-databricks,!snapshot-shims' clean verify -Dpytest.TEST_TAGS=''
+mvn -U -B $MVN_URM_MIRROR '-Pinclude-databricks,!snapshot-shims' clean verify -Dpytest.TEST_TAGS='not cudf_udf'
```

Comment:
I think we can keep it as `''` here since, otherwise it will overwrite the default `TEST_TAGS ` and try to run the qarun as per https://github.com/NVIDIA/spark-rapids/pull/640/files#diff-600376dffeb79835ede4a0b285078036?

Reply:
as talked offline, we could also turn off qurun probably in another diff